core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -111,15 +113,15 @@ public Application(T config) {
 
   public Application(T config, String path) {
     this.config = config;
-    this.path = Objects.requireNonNull(path);;
+    this.path = Objects.requireNonNull(path);
 
     this.metrics = configureMetrics();
     this.getMetricsTags().putAll(
             parseListToMap(config.getList(RestConfig.METRICS_TAGS_CONFIG)));
 
-    this.requestLog = new Slf4jRequestLog();
-    this.requestLog.setLoggerName(config.getString(RestConfig.REQUEST_LOGGER_NAME_CONFIG));
-    this.requestLog.setLogLatency(true);
+    Slf4jRequestLogWriter logWriter = new Slf4jRequestLogWriter();
+    logWriter.setLoggerName(config.getString(RestConfig.REQUEST_LOGGER_NAME_CONFIG));
+    requestLog = new CustomRequestLog(logWriter, CustomRequestLog.EXTENDED_NCSA_FORMAT + " %msT");
```

Comment:
Similar change in Apache Kafka: https://github.com/apache/kafka/pull/6668/files#diff-9c19b57e1c70b306f5bad03f18d656321289523e0d50f975863ebfcb6b7095c0R223